sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.2.0-beta.1 (Unreleased)
 
```

Comment:
I guess we missed it in the previous PR, we need to add the service bump version

Reply:
Updated.

Reply:
Noticed my push didn't work and the changelog was not updated. Adding the change to the next PR.